TBODeveloperOverlay/TBODeveloperOverlay/InspectorCollection.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  InspectorCollection.swift
+//  TBODeveloperOverlay
+//
+//  Created by Pascal Stüdlein on 06.10.17.
+//  Copyright © 2017 TBO Interactive GmbH & CO KG. All rights reserved.
+//
+
+import Foundation
+
+public class InspectorCollection {
+    public static let defaultCollection: InspectorCollection = InspectorCollection(inspectors: [NumberInspectorViewController.self, BoolInspectorViewController.self])
+    
+    private let inspectors: [InspectorViewController.Type]
+    
+    init(inspectors: [InspectorViewController.Type]) {
```

Comment:
This should be a public init, right? Just in case one wants to implement their own?

Reply:
(y)